packages/react/src/callout/CalloutTitle.tsx

```diff
@@ -1,12 +1,23 @@
 import { PropsWithChildren } from 'react';
 import { Text } from '../text';
+import { CalloutVariant, calloutVariantMap } from './utils';
 
 export type CalloutTitleProps = PropsWithChildren<{
 	as?: 'h2' | 'h3' | 'h4' | 'h5';
+	variant: CalloutVariant;
```

Comment:
This is a new *required* prop that consumers could be using so this is technically a breaking change. If you make it optional, and default to `neutral` it should be fine 👍